kyuubi-server/src/main/scala/org/apache/kyuubi/engine/ApplicationOperation.scala

```diff
@@ -58,7 +58,7 @@ trait ApplicationOperation {
    * @param tag the unique application tag for engine instance.
    * @return [[ApplicationInfo]]
    */
-  def getApplicationInfoByTag(tag: String): ApplicationInfo
+  def getApplicationInfoByTag(tag: String, submitTime: Option[Long] = None): ApplicationInfo
```

Comment:
can we add a new method with the default implementation ?
```scala
def getApplicationInfoByTag(tag: String, submitTime: Long) = getApplicationInfoByTag(tag)
```

Reply:
I don't quite understand what you mean by using
```scala
def getApplicationInfoByTag(tag: String): ApplicationInfo = getApplicationInfoByTag(tag, None)
```
to support default implementation?

Reply:
you can have two methods:
```scala
def getApplicationInfoByTag(tag: String, submitTime: Long): ApplicationInfo =
  getApplicationInfoByTag(tag)

def getApplicationInfoByTag(tag: String): ApplicationInfo
```

Reply:
compared two approaches, this one is clean and does not touch many things